config/default.yaml

```diff
@@ -48,7 +48,7 @@ services:
   draftStore:
     redis:
       host: 'localhost'
-      port: 6380
+      port: 6379
```

Comment:
Please exclude this redis port change.

Reply:
This change can stay however it is not recommended to be committed.